app/src/main/kotlin/com/wire/android/ui/userprofile/avatarpicker/AvatarPickerViewModel.kt

```diff
@@ -63,21 +68,22 @@ class AvatarPickerViewModel @Inject constructor(
             withContext(dispatchers.io()) {
                 pictureState = avatarImageManager
                     .postProcessAvatar(imageUri)
-                    ?.let { PictureState.Picked(it) } ?: PictureState.Empty
+                    ?.let { PictureState.Picked(it) } ?: PictureState.Picked(imageUri)
             }
         }
     }
 
-    fun uploadNewPickedAvatarAndBack() {
+    fun uploadNewPickedAvatarAndBack(context: Context) {
```

Comment:
From my side it's ok to pass the `Context` here, not the best option in terms of testability, but we can improve it later 💪